snippet-extractor-output/snippets.yaml

```diff
@@ -2500,6 +2500,139 @@
 
         await context.sync();
     });
+'Excel.FormattedNumberCellValue#basicType:member':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/excel/85-preview-apis/data-types-formatted-number.yaml
+
+    await Excel.run(async (context) => {
+      // This method creates a formatted number data type,
+      // and sets the format of this data type as a date.
+
+      // Create a worksheet called "Sample" and activate it.
+      context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+      const sheet = context.workbook.worksheets.add("Sample");
+      sheet.activate();
+      
+      // Get cell A1 and write a number formatted as a date to cell A1.
+      const dateRange = sheet.getRange("A1");
+      dateRange.valuesAsJson = [
+        [
+          {
+            type: Excel.CellValueType.formattedNumber,
+            basicValue: 32889.0,
+            numberFormat: "m/d/yyyy"
+          }
+        ]
+      ];
+      await context.sync();
+    });
+'Excel.FormattedNumberCellValue#basicValue:member':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/excel/85-preview-apis/data-types-formatted-number.yaml
+
+    await Excel.run(async (context) => {
+      // This method creates a formatted number data type,
+      // and sets the format of this data type as a date.
+
+      // Create a worksheet called "Sample" and activate it.
+      context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+      const sheet = context.workbook.worksheets.add("Sample");
+      sheet.activate();
```

Comment:
Moving the worksheet creation/deletion code will also take it out of the ref doc sample.